NotesBackEnd/src/Notes.Api.Services/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("Notes.Api.Services")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]
+[assembly: AssemblyProduct("Notes.Api.Services")]
+[assembly: AssemblyCopyright("Copyright ©  2017")]
+[assembly: AssemblyTrademark("")]
+[assembly: AssemblyCulture("")]
+
+[assembly: InternalsVisibleTo("Notes.Api")]

```

Comment:
No, no, no, no 😉 
Only use `InternalsVisibleTo` for tests or legacy code you cannot refactor.

There is another way to provide the `Container` with the implementation of an `API` specific data class (or more precisely instance). Discuss this with Vašek or me, please.